packages/legend-application-repl/src/stores/dataCube/DataCubeStore.tsx

```diff
@@ -14,94 +14,46 @@
  * limitations under the License.
  */
 
-import type { LegendREPLApplicationStore } from './LegendREPLBaseStore.js';
-import { REPLServerClient } from '../server/REPLServerClient.js';
 import {
-  ActionState,
-  assertErrorThrown,
-  guaranteeNonNullable,
-  LogEvent,
-  NetworkClient,
-} from '@finos/legend-shared';
-import { makeObservable, observable } from 'mobx';
-import { DataCubeState } from './dataCube/DataCubeState.js';
-import { DataCubeEngine } from './dataCube/DataCubeEngine.js';
-import { LicenseManager } from '@ag-grid-enterprise/core';
-import { ActionAlertType, APPLICATION_EVENT } from '@finos/legend-application';
+  ActionAlertType,
+  APPLICATION_EVENT,
+  type GenericLegendApplicationStore,
+} from '@finos/legend-application';
+import { type AbstractDataCubeEngine } from './DataCubeEngine.js';
+import { DataCubeState } from './DataCubeState.js';
 import {
   DEFAULT_SMALL_ALERT_WINDOW_CONFIG,
+  DisplayState,
   LayoutConfiguration,
   LayoutManagerState,
-  DisplayState,
   WindowState,
   type WindowConfiguration,
-} from './LayoutManagerState.js';
+} from '../LayoutManagerState.js';
+import { DocumentationPanel } from '../../components/repl/DocumentationPanel.js';
+import { SettingsPanel } from '../../components/repl/SettingsPanel.js';
+import {
+  ActionState,
+  assertErrorThrown,
+  guaranteeNonNullable,
+  LogEvent,
+} from '@finos/legend-shared';
+import { LicenseManager } from '@ag-grid-enterprise/core';
 import {
   Alert,
   AlertType,
   type AlertAction,
-} from '../components/repl/Alert.js';
-import { DocumentationPanel } from '../components/repl/DocumentationPanel.js';
-import { SettingsPanel } from '../components/repl/SettingsPanel.js';
+} from '../../components/repl/Alert.js';
 
-export class REPLStore {
-  readonly application: LegendREPLApplicationStore;
-  readonly client: REPLServerClient;
+export class WindowNotificationManager {
```

Comment:
these alert methods could potentially do more than just opening new window for error/warning, so let's put them back at the level of `DataCubeStore`. Thanks!